packages/inventory/README.md

```diff
@@ -74,3 +60,75 @@ const props = {
   - Route to the service point profile view to redirect to after pressing `Cancel` button on the form to abort and exit the view or after successful submission.
   - If not provided the default is `/inventory/profile/<id>` where `id` is the id of the service point
   - If provided, the id of the service point will be added to the route name e.g if you passed `/inventory/service-point` as the `servicePointProfileURL`, the redirect will be to `/inventory/service-point/<id>`
+  -
+
+### Inventory List
+
+To add a view for inventory list:
+
+```tsx
+import { ConnectedServicePointList } from '@opensrp/inventory';
+
+const props = {
+	openSRPBaseURL: 'https://opensrp-stage.smartregister.org/opensrp/rest',
+}
+
+<Route path={`/inventory/list`}>
+  <ConnectedServicePointList {...props} />
+</Route>
+```
+
+`ConnectedServicePointList` component takes the following props:
+
+- **opensrpBaseURL:**(string)
+
+  - **required**
+  - Opensrp API base URL
+
+### Service Point Profile
+
+To add a view for service point profile:
+
```

Comment:
I find adding screenshot of the component after the description really complementary. @haseemVD What do you think?